packages/@rnw-scripts/integrate-rn/package.json

```diff
@@ -17,7 +17,7 @@
     "@rnw-scripts/package-utils": "^0.0.3",
     "async": "^3.2.0",
     "lodash": "^4.17.15",
-    "ora": "^4.0.3",
+    "ora": "^3.4.0",
```

Comment:
>3. [](start = 13, length = 2)

This is a downgrade from semantic version point of view, why?

Reply:
We had a few different versions of ora in our repo. The latest typings available are for 3.0, so I changed packages to use that consistently.